contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -216,11 +216,6 @@ mod SequencerUptimeFeed {
         AccessControl::has_access(user, data)
     }
 
-    #[view]
-    fn check_access(user: ContractAddress) {
```

Comment:
1. this was removed because rather than relying on an external call to a view function to revert, I figured it would be better practice and more robust to read data from external contacts and handle the revert internally
2. access_control.cairo is the library, intended to be imported and used as such, e.g. DAO contract inherits access control functionality and uses it to assign management roles. access_contoller.cairo (being interacted with here) is the deployable contract version and doesn't expose check_access as an external function, for the reason above.
3. yep this seems to be the only place we interact with a deployed access controller, but will double check. haven't been updating the hardgat tests, but I do have another pr in progress to reimplement in Cairo if we want to replace them moving forwards

Reply:
1. I did think it's a bit unclear if a library function reverts or not because the name doesn't specify if a bool is returned or if the function panics. We could make it clearer by prefixing with `assert_`, eg` assert_has_access` but for now I think the change is OK 👍🏻